### DISCUSSION TITLE
Add support for privacy-enhanced youtube to video shortcode

### DISCUSSION THREAD

**Discussion by jmgirard** [quarto-dev/quarto-cli#4050] [Feature Requests]
When you share a video from youtube with "privacy-enhanced mode" enabled, the link will be `youtube-nocookie.com` instead of `youtube.com`. It would be great to support this in the video shortcode:

```qmd
{{< video https://www.youtube-nocookie.com/embed/mrvhj2XUfWo >}}
```

Currently (v1.3.124), you get a "No compatible source was found for this media" error.

**Comment by cderv** [comment]
It seems like something we can detect. I have opened https://github.com/quarto-dev/quarto-cli/issues/4060

**Reply by cderv** [reply]
@jmgirard It should be fixed in next pre-release version build probably later today

**Comment by mcanouil** [comment]
Closing this as completed and in favour of https://github.com/quarto-dev/quarto-cli/issues/4060
